packages/coding-standard/src/Rules/AbstractCheckFunctionCallRule.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Rules;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use Symplify\CodingStandard\PhpParser\NodeNameResolver;
+
+abstract class AbstractCheckFunctionCallRule extends AbstractSymplifyRule
```

Comment:
This name looks too generic.
Maybe `AbstractPrefferedCallOverFuncRule` is more explicit, also is more alike to children names

Reply:
implemented.